tests/spread/general/scriptlets/task.yaml

```diff
@@ -0,0 +1,30 @@
+summary: Validate scriptlet failures
+
+environment:
+  SNAP_DIR: scriptlet-failures
+
+prepare: |
+  #shellcheck source=tests/spread/tools/snapcraft-yaml.sh
+  . "$TOOLS_DIR/snapcraft-yaml.sh"
+  set_base "$SNAP_DIR/snap/snapcraft.yaml"
+
+restore: |
+  #shellcheck source=tests/spread/tools/snapcraft-yaml.sh
+  . "$TOOLS_DIR/snapcraft-yaml.sh"
+  cd "$SNAP_DIR"
+
+  snapcraft clean
+  rm -f ./*.snap
+
+  restore_yaml "snap/snapcraft.yaml"
+
+execute: |
+  #shellcheck source=tests/spread/tools/snapcraft-yaml.sh
+  . "$TOOLS_DIR/snapcraft-yaml.sh"
+  cd "$SNAP_DIR"
+
+
```

Comment:
random question about the amount of empty lines and the thought process put into it

Reply:
There does seem to be an extra line there for no reason. Just an accident.
```suggestion
```